models/bob.js

```diff
@@ -22,18 +23,43 @@ const BobModel = mongoose.model('Bob', bobSchema);
   @param {Object[]} bobData
 */
 function saveBob(bobData) {
+  let mediaStatus = "";
+  if(bobData.data.Link) {
+    mediaStatus = false;
+  } else {
+    console.log("Warning: Saving bob without Link");
```

Comment:
Right now, all bobs should have media, because that is the only kind of bob that exists. Before adding new bob types back in, we should restructure the db model to reflect what we want it to hold.